packages/amplify-graphiql-explorer/src/App.tsx

```diff
@@ -299,9 +312,26 @@ class App extends Component<{}, State> {
         }}
       />
     ) : null;
+
+    const clearDataModal = clearDataModalVisible ? (
+      <ClearDataModal
+        onClose={() => {
+          this.setState({ clearDataModalVisible: false });
+        }}
+        onClear={async () => {
+          this.setState({ clearDataModalVisible: false });
+          this.setState({ clearResponse: JSON.stringify(await clearData(), undefined, 2) });
+          setTimeout(() => {
+            this.setState({ clearResponse: '' });
+          }
+          , 8000);
+        }}
```

Comment:
I'd pull these methods out into their own definition, just to separate the functionality and binding to the component here.

Also, where's the 8s delay coming from?

Reply:
It was an arbitrary delay. The message was still showing up even after running other queries. So after 8 seconds the deletion message gets cleared